morepath/authentication.py

```diff
@@ -36,42 +32,10 @@ class IdentityPolicyRegistry(object):
     """Register the current identity policy.
```

Comment:
In my dispatchmethods-decoupled branch I can do away with this registry entirely, simply overwriting methods on the app class directly. (thanks to ``app_class_arg``). Though haven't read your ``directive.py`` yet, it may be this strategy allows for an ``app`` argument.